mods/tnt/init.lua

```diff
@@ -2,9 +2,6 @@ tnt = {}
 
 -- Default to enabled when in singleplayer
 local enable_tnt = minetest.setting_getbool("enable_tnt")
```

Comment:
Needs to default to true in the absence of the setting. 'getbool' returns 'nil' if the setting is not present.